### DISCUSSION TITLE
Display name for document types and element types

### DISCUSSION THREAD

**Discussion by royberris** [umbraco/Umbraco-CMS#11646] [Features and ideas]
When using element types picker we can specify a "Template" for each element type. This template shows up as a display name under the Content section. This works great because we have Dutch content editors, but the developers write the code and settings stuff in English. 

The created elements show the name from the "Template" field. But when you create a new element, in the create dialog it still shows the element type name. I would like to see this also being the "Template" field for consistency. Also it is confusing when Dutch editors work with the Dutch template names, but English element type names.

Now I want to extend this to document types. I want to be able to name my document type "Contact Page" (English) but for my editors when they create a new document type I would like it to be in Dutch. Where it will say a "Display name" variant of the document type.

Especially for more technical naming, something we might have a long technical name like "Automatic Content Carousel Page Component". But for a editor this would just be a "Automatische Carousel".

-----

How would this work? Maybe we could have an optional display name that Umbraco uses for the backoffice when users are under the Content / Media TAB. Or maybe for specific groups?

**Comment by rasmusjp** [comment]
This is already possible using dictionary items, I couldn't find any official documentation on it but I did find this old article explaining how it works https://24days.in/umbraco-cms/2013/the-dictionary-secrets/ and it still works with Umbraco 7-9
